roles/opnsense_configure/meta/main.yml

```diff
@@ -0,0 +1,9 @@
+galaxy_info:
+  author: Fabio Bertagna <bertagna@puzzle.ch>
+  company: Puzzle ITC
+  license: GPL-3.0-only
+  min_ansible_version: 2.1
+  galaxy_tags:
+    - opnsense
+
+dependencies: [ ]
```

Comment:
wound't the module be dependency of the role

Reply:
My understanding of the `dependency` block in this file is that you only need to list dependency to other roles. https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#dependencies